.github/workflows/e2e-tests.yml

```diff
@@ -183,6 +185,14 @@ jobs:
         env:
           TERM: xterm
 
+      - name: Upload Replay.io recordings
+        if: failure()
```

Comment:
```suggestion
```

Reply:
Let's make this into if: always(), just so it doesn't bail when the previous step fails.